docs/examples/201/vm-copy-managed-disks/main.bicep

```diff
@@ -0,0 +1,191 @@
+param virtualMachineAdminUserName string = 'azadmin'
+param virtualMachineAdminPassword string {
```

Comment:
need to switch to decorators syntax for all parameter modifiers - so this would change to:

```bicep
@secure()
param virtualMachineAdminPassword string
```

Reply:
Is this new in 0.3.1?

Reply:
yep